htdocs/config.php

```diff
@@ -1,5 +1,5 @@
 <?php
-@date_default_timezone_set(@date_default_timezone_get());
+ini_set('date.timezone', 'UTC');
```

Comment:
This will change server time from CET (using daylight saving) to always being UTC (no daylight savings), which will change all the in-game time stuff

Reply:
Indeed. For some reason I checked last night and thought they were on the same timezone. Do you have an objection to using UTC in principle, or just in the middle of a game? (Also, won't this only affect the time display, since `TIME` should be independent of timezone.)

Reply:
I've got no problem with UTC in principle, it's just that after all these years a lot of players will be used to server time as it currently stands so the change will be a little confusing - and having it in the middle of the game would be even more confusing, so keeping it between games and with an announcement would help a lot.  Also whilst it won't affect game mechanics it will affect the time display in the top left as that's based off of the php timezone right? And what about the time offset in prefs, I don't remember if that is offset against UTC or against server time (I guess realistically it should be timezones rather than a flat offset in order to account for DST, but that might be quite a headache to handle)